dataset/nigeria_kaduna.yaml

```
@@ -0,0 +1,3 @@
+country: 'NG' 
```

Not sure if it's important, but for the other datasets the country is in lowercase.

```suggestion
country: 'ng' 
```

---

Fair! For our information, the [ISO codes are capitalized](https://www.iso.org/obp/ui/#search/code/)